Flick/Controllers/ListViewController.swift

```diff
@@ -68,22 +70,51 @@ class ListViewController: UIViewController {
         mediaCollectionView.bounces = false
         view.addSubview(mediaCollectionView)
 
+        // Empty list
+        addMediaMessageLabel.text = "Nothing here yet. Add\nsome movies or shows!"
+        addMediaMessageLabel.textColor = .darkBlue
+        addMediaMessageLabel.textAlignment = .center
+        addMediaMessageLabel.font = .systemFont(ofSize: 18, weight: .medium)
+        addMediaMessageLabel.numberOfLines = 0
+
+        arrowToAddButtonView.image = UIImage(named: "arrowToButton")
+        emptyListImageView.image = UIImage(named: "emptyList")
+
+        if list.shows.count == 0 {
+            view.addSubview(emptyListImageView)
+            view.addSubview(addMediaMessageLabel)
+            view.addSubview(arrowToAddButtonView)
+        }
+
         setupSections()
         setupConstraints()
     }
 
     init(list: MediaList) {
         super.init(nibName: nil, bundle: nil)
         self.list = list
-        listNameLabel.text = list.lstName
     }
 
     required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
 
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+
+        if list.shows.count != 0 {
```

Comment:
is there a need to render these at all if there are no movies? instead of hiding, can we just only initialize those views if in empty state and if not, render the other views?